tembo-cli/tembo/docker-compose.yml.template

```diff
@@ -17,6 +17,28 @@ services:
       - "traefik.tcp.routers.{{instance.instance_name}}.entrypoints=postgresql"
       - "traefik.tcp.routers.{{instance.instance_name}}.tls.passthrough=true"
       - "traefik.tcp.services.{{instance.instance_name}}.loadbalancer.server.port=5432"
+
+  {% if instance.app_services.restapi %}
+  {{instance.instance_name}}-postgrest:
+    image: postgrest/postgrest:v10.0.0
```

Comment:
Im assuming in the future, these local apps will just run the docker images that we use for cloud?

Reply:
I can change that. What image does cloud use for `postgrest`? Also, I will make it configurable in future PRs.

Reply:
All app's yaml files are in the control-plane repo at the moment. We should find a way to put them, and the Stack definitions, into a stand-alone crate IMO.

https://github.com/tembo-io/control-plane/blob/293b6f1bf11addd13d7bd66f969d9392d494f7a6/cp-webserver/src/apps/restapi.yaml#L3